tp-test/run.go

```diff
@@ -139,7 +139,7 @@ func runABTest(ctx context.Context, failed chan struct{}, opts runABTestOptions)
 		store.SetTest(t.ID, TestPassed, "")
 
 		conn1.ExecContext(ctx, "drop database if exists "+dbName1)
```

Comment:
Should we comment this line? Sometimes we need to check the sync result and the data should not be deleted.